python2/cloudwatch/README.md

```diff
@@ -0,0 +1,89 @@
+# CloudWatch Logs Shipper - Lambda
```

Comment:
Consult Shalom, maybe it's worth to add another comment here about not supporting it in the near future?